.github/workflows/eerelease.yml

```
@@ -97,18 +97,23 @@ jobs:
       - name: Login to the Github Container registry
         env:
           REGISTRY_USER: ${{ github.actor }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         uses: redhat-actions/podman-login@v1
         with:
           registry: ghcr.io/${{ github.repository_owner }}
-          username: ${{ github.actor }}
-          password: ${{ github.token }}
+          username: $REGISTRY_USER
+          password: $GITHUB_TOKEN
```

That does not work. You have to use `${{ env.xxx }}` instead of `$xxx`. But there's no reason to use `env` at all anyway, that's only potentially exposing environment variables to parts of the action which shouldn't have access to it.
```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

---

Yes, my bad. There is a reason that one should not work when you are awake for 24 hours +. :)